addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -62,10 +62,21 @@ database, without the s_newsletter_list class. See fixNewsletterListClass.
                     <div class="mb-0 py-2 col-12 s_website_form_field s_website_form_model_required" data-type="char" data-name="Field">
                         <div class="row s_col_no_resize s_col_no_bgcolor">
                             <label class="col-form-label col-sm-auto s_website_form_label" style="width: 250px" for="mailing_sub1">
-                                <span class="s_website_form_label_content">Your Name</span>
+                                <span t-if="request.env['mailing.contact']._is_name_split_activated()" class="s_website_form_label_content">Name</span>
+                                <span t-else="" class="s_website_form_label_content">Your Name</span>
```

Comment:
Let's align those 2, it really does not make much of a difference, in both case we want "your name" as in the name of the customer.